### PR TITLE
Quarterly Gas Rebate Distribution August 2023 - Review

### PR DESCRIPTION
**Pull request by marczeller**

### CODE REVIEW

File: src/20230906_AaveV3_Eth_QuarterlyGasRebateDistributionAugust2023/AaveV3_Ethereum_QuarterlyGasRebateDistributionAugust2023_20230906.sol [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+      AaveV2EthereumAssets.WETH_UNDERLYING,
+      address(this),
+      TOTAL_AMOUNT
+    );
+
+    // convert WETH to ETH
+    IWETH(WETH).withdraw(TOTAL_AMOUNT);
+
+    // transfer ETH to delegates
+    payable(ACI).transfer(ACI_AMOUNT);
```
Comment (brotherlymite) [inline src/20230906_AaveV3_Eth_QuarterlyGasRebateDistributionAugust2023/AaveV3_Ethereum_QuarterlyGasRebateDistributionAugust2023_20230906.sol:58]: @marczeller using transfer is not recommended (more about it in this [article](https://consensys.io/diligence/blog/2019/09/stop-using-soliditys-transfer-now/)) and was the reason the payload execution was failing in the tests.
Using openzeppelin `sendValue()` should solve it.

Comment (brotherlymite) [reply src/20230906_AaveV3_Eth_QuarterlyGasRebateDistributionAugust2023/AaveV3_Ethereum_QuarterlyGasRebateDistributionAugust2023_20230906.sol:58]: You can just do the following to fix it:

```
import {Address} from 'solidity-utils/contracts/oz-common/Address.sol';

contract AaveV3_Ethereum_QuarterlyGasRebateDistributionAugust2023_20230906 is
  IProposalGenericExecutor
{
  using Address for address payable;
  
  ...
  
  payable(ACI).sendValue(ACI_AMOUNT);
  payable(HKUST).sendValue(HKUST_AMOUNT);
  payable(TOKEN_LOGIC).sendValue(TOKEN_LOGIC_AMOUNT);
  payable(MICHIGAN).sendValue(MICHIGAN_AMOUNT);
  payable(LBS).sendValue(LBS_AMOUNT);
  payable(WINTERMUTE).sendValue(WINTERMUTE_AMOUNT);
  payable(KEYROCK).sendValue(KEYROCK_AMOUNT);
  payable(STABLELAB).sendValue(STABLELAB_AMOUNT);
}
```

Comment (marczeller) [reply src/20230906_AaveV3_Eth_QuarterlyGasRebateDistributionAugust2023/AaveV3_Ethereum_QuarterlyGasRebateDistributionAugust2023_20230906.sol]: thx! fix implemented.

File: src/20230906_AaveV3_Eth_QuarterlyGasRebateDistributionAugust2023/AaveV3_Ethereum_QuarterlyGasRebateDistributionAugust2023_20230906.sol [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+      AaveV2EthereumAssets.WETH_UNDERLYING,
+      address(this),
+      TOTAL_AMOUNT
+    );
+
+    // convert WETH to ETH
+    IWETH(WETH).withdraw(TOTAL_AMOUNT);
+
+    // transfer ETH to delegates
+  payable(ACI).sendValue(ACI_AMOUNT);
```
Comment (brotherlymite) [inline src/20230906_AaveV3_Eth_QuarterlyGasRebateDistributionAugust2023/AaveV3_Ethereum_QuarterlyGasRebateDistributionAugust2023_20230906.sol:61]: @marczeller can you also run prettier once to fix the spaces.

**Review by rustboyar — APPROVED** [review]

**Review by rustboyar — DISMISSED** [review]

**Review by rustboyar — APPROVED** [review]

**Pull request merged**